sys/fido2/ctap/ctap.c

```diff
@@ -192,16 +192,6 @@ static int _save_pin(uint8_t *pin, size_t len);
  */
 static int _write_state_to_flash(const ctap_state_t *state);
```

Comment:
Though it is not directly related to the PR, I still think it would improve the code if you rename `_save_rk` to be in line with this function naming scheme (as [commented before](https://github.com/RIOT-OS/RIOT/pull/17096#discussion_r833110378))